.github/linter/customRules.ts

```diff
@@ -281,13 +282,14 @@ export const metadataStatusIsValid = {
 
     const validStatus = [
       "Idea",
-      "Draft", 
       "Review",
       "Accepted",
       "Stagnant",
       "Withdrawn",
       "Implemented",
       "Activated",
+      "Stale",
```

Comment:
What's the difference between `Stagnant` and `Stale`? We have `Stagnant` listed in simd-1

Reply:
ahh yes, good call

Reply:
fixed